hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -877,7 +877,7 @@ def try_shrinking_blocks(self, blocks, b):
                 blocks = blocks[:i]
                 break
             u, v = self.blocks[block].bounds
-            n = min(v - u, len(b))
+            n = min(self.blocks[block.length], len(b))
```

Comment:
→ `self.blocks[block].length`

That should make the tests happy again.

Reply:
Whoops, thanks.